base_comment_template/views/base_comment_template_view.xml

```diff
@@ -26,7 +26,6 @@
                 <field name="partner_ids" />
                 <field name="model_ids" />
                 <field name="domain" />
-                <field name="priority" />
```

Comment:
Shouldn't this be added somewhere with the new name?

Reply:
`sequence` field it not show in search view or form view, only in tree view with `widget="handle"` right?

Reply:
You can show it also as number in form. Some objects do that for a direct assignation.